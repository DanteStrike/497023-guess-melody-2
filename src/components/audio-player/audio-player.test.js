import React from "react";
import renderer from "react-test-renderer";
import AudioPlayer from "./audio-player.jsx";

it(`Should render correctly AudioPlayer component`, () => {
  const component = renderer
    .create(
        <AudioPlayer
          isActivePlayer={false}
          src={`http://dl2.mp3party.net/online/9116246.mp3`}
          renderPlayButton={jest.fn()}
          audioRef={jest.fn()}
        />
    ).toJSON();

  expect(component).toMatchSnapshot();
});
