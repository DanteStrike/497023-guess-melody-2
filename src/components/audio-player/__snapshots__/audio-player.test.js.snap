// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly AudioPlayer component 1`] = `
<div
  className="track__status"
>
  <audio
    src="http://dl2.mp3party.net/online/9116246.mp3"
  />
</div>
`;
